Model/User.hs

```diff
@@ -29,3 +31,6 @@ findUsers userIds = selectList [UserId <-. userIds] []
 -- | Same as @findUsers@, but discards the @entityKey@
 findUsers' :: [UserId] -> DB [User]
 findUsers' = fmap (map entityVal) . findUsers
+
+disableCreateSite :: Int -> User -> Bool
```

Comment:
Maybe `hasReachedSiteQuota` or something so that the `Int` argument here is more obvious?